css/modules.css

```diff
@@ -1,67 +1,62 @@
 /*----------------Header--------------------*/
-.header_nav li {
+.main_header a {
   text-transform: uppercase;
-  list-style: none;
-  display:inline-block;
-  padding: 0px;
-  border: 2px;
-  font-size: 1em;
+  text-decoration: none;
+  color: black;
 }
-.top {
+  /*padding: 3%;
```

Comment:
If you're not using it, delete it. No commented out code in PR's, that's why we have version control.